src/test/__tests__/components/PreviewExternalResourcePane.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { PreviewExternalResourcePane } from '@components/PreviewExternalResourcePane';
+import state from '@state';
+import { fireEvent, screen } from '@testing-library/dom';
+import { render } from '@testing-library/react';
+import { RecoilRoot } from 'recoil';
+
+const navigate = jest.fn();
+const getRecordTitle = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => navigate,
+}));
+
+jest.mock('@common/helpers/record.helper', () => ({
```

Comment:
```
jest.mock('@common/helpers/record.helper', () => ({
  getRecordTitle,
}));
```

Reply:
Doesn’t work like this for some reason… Might have something to do with how jest works (hoists) stuff